M5StickC-master/src/AXP192.cpp

```diff
@@ -523,6 +523,21 @@ void AXP192::SetLDO3(bool State)
     Write1Byte( 0x12 , buf );
 }
 
+void AXP192::SetGPIO0(bool State)
```

Comment:
We should probably update the onboarding steps/notes so that developers would use this custom branch of M5Stick-C library instead of the public version.
Their AXP192 lib is missing many apis.

Reply:
Yeah I'm sure we'll slowly implement more and more stuff as we go along